lib/generators/fuel/views_generator.rb

```diff
@@ -9,7 +9,7 @@ class ViewsGenerator < Rails::Generators::Base
       source_root File.expand_path('../../../../app/views', __FILE__)
 
       def copy_views
-        directory 'fuel', 'app/views/fuel/posts'
+        directory 'fuel', 'app/views/fuel'
```

Comment:
@reubenbrown thanks for the PR and good catch. I wonder if it would be a better idea to remove the admin views from the generator altogether? If anything it makes upgrading more difficult with antiquated admin views out there in the wild.